### ISSUE TITLE
Error on screen after running 'drush cr'

### ISSUE THREAD

**Issue by alanburke** [computerminds/parrot#58]
Hi Steven
I'm testing Drupal 8, with php5.4 after the recent update - thanks!

Not sure where this issue lies, but Parrot won the short straw, so I'll start here.
I get an error on screen after running drush cr [on the host - Mac OS 10.9.1 ]

```
Twig_Error_Loader: The "/Users/alanjosephburke/projects/parrot54/sites/imp.parrot" directory does not exist. in Twig_Loader_Filesystem->addPath() (line 93 of /vagrant_sites/imp.parrot/core/vendor/twig/twig/lib/Twig/Loader/Filesystem.php).
```

The error report the host file system as  path it was checking.
See attached for full error.
![drush-cr-parrot](https://f.cloud.github.com/assets/286566/2304174/7d6b5896-a20a-11e3-9f96-5b122d1ebe56.jpg)

If I run drush cr from the VM, I don't get the error.

Any tips on where the error might lie. 
Thanks for all the hard work.

**Comment by googletorp** [comment]
I don't think this is relevant anymore, and don't think we can do anything with parrot to solve this.

If you don't want to connect to the VM when running drush, you can create a drush alias which does this behind the scene.
